apps/src/templates/rubrics/RubricSettings.jsx

```diff
@@ -171,6 +261,24 @@ export default function RubricSettings({
     });
   };
 
+  const handleRunAiAssessmentAll = () => {
+    setStatusAll(STATUS_ALL.EVALUATION_PENDING);
+    const url = `/rubrics/${rubricId}/run_ai_evaluations_for_all`;
+    const params = {section_id: sectionId};
+    fetch(url, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+        'X-CSRF-Token': csrfToken,
+      },
+      body: JSON.stringify(params),
+    }).then(response => {
+      if (!response.ok) {
+        setStatus(STATUS_ALL.ERROR);
```

Comment:
I see you are just copying the existing pattern, but i am wondering if there could be a race condition here -- when this status message is displayed, it seems like it could soon be overwritten by whatever status is set by the 5-second polling. I think this is ok, just noting for future debugging in case we run into issues.